src/serverMain/kotlin/com/studo/campusqr/endpoints/UserFrontend.kt

```diff
@@ -14,17 +12,15 @@ import java.util.*
 
 
 suspend fun ApplicationCall.userFrontend() {
-  val locationId = parameters["l"]
-  val locationName = locationId?.let { id -> getLocationOrNull(id)?.name }
-  val language = this.language
+  val fullLocationId = parameters["l"]
+  var seatNr: Int? = null
```

Comment:
you call it here `seatNr`, on other occurences it is called just `seat`. We should stick to one naming if it is the same.